src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp

```diff
@@ -1334,4 +1338,4 @@ void AddSC_boss_freya()
     new achievement_freya_knock_on_wood("achievement_freya_knock_on_wood", 1);
     new achievement_freya_knock_on_wood("achievement_freya_knock_knock_on_wood", 2);
     new achievement_freya_knock_on_wood("achievement_freya_knock_knock_knock_on_wood", 3);
-}
+}
```

Comment:
Remove the white space at the end

Reply:
Not sure how to fix that, see #welcome-n-offtopic on discord if you can. There is no space in my file.

Reply:
My bad on this one, was because of the icon from github!

Reply:
i think there was an issue indeed, locus313 said on discord i need a new line at the end of the file. after pushing that, the difference vanished :-)